build/projects/Add-Ons (W1)/.AL-Go/PreCompileApp.ps1

```diff
@@ -0,0 +1,20 @@
+Param(
+    [string] $appType,
+    [ref] $compilationParams
+)
+
+if ($ENV:BuildMode -eq 'Clean') {
+
+    # If useProjectDependencies is set to false, we need to remove the app symbols folder so we recompile everything.
+    $settings = Get-Content (Join-Path $PSScriptRoot "settings.json" -Resolve) | ConvertFrom-Json
```

Comment:
Aren't AL-Go settings (`$env:settings`) available here?

Reply:
Good point. It should be. I'll use that instead